python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -601,8 +601,14 @@ def get_nodes_for(node_types: Dict[NodeType, NodeTypeConfigDict],
             # starts up because placement groups are scheduled via custom
             # resources. This will behave properly with the current utilization
             # score heuristic, but it's a little dangerous and misleading.
-            logger.info(
-                "No feasible node type to add for {}".format(resources))
+            logger.warning(
+                f"The autoscaler could not find a node type to satisfy the"
+                f"request: {resources}. If this request is related to "
+                f"placementt groups the resource request will resolve itself, "
```

Comment:
kinda, it's not quite as simple as `"pg" in resource` because you need to figure out if `{"Custom1": 1, "placement_group_string": 1}` is infeasible because of the placement group or because there are no nodes with the "Custom1" resource.